doc/overlays.xml

```diff
@@ -96,4 +96,68 @@ overridden and/or new packages.</para>
 
 </section>
 
+<section xml:id="sec-overlays-extend-modify">
```

Comment:
Hm, the instructions in this section seem more complicated to me than what has already been recommended in the preceding section. The manual has already stated how to add new packages or modify existing ones. Maybe turn this into an "extended example" or at least clarify how it is different than what precedes it.